service/src/main/resources/logback.xml

```diff
@@ -47,6 +47,7 @@ As such, enabling logback status data is very highly recommended and should be c
 
     <!-- skip info logs for unmapped routes -->
     <logger name="spark.http.matching.MatcherFilter" level="WARN"/>
+    <logger name="com.commercetools.pspadapter.payone.IntegrationService" level="debug"/>
```

Comment:
Lets try to make it configurable today!!

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/331/commits/e612747250e87e092ecd44eee23805893d03ea1c